src/polyswarm/metadata.py

```diff
@@ -0,0 +1,20 @@
+import click
+
+
+@click.group(short_help='Interact with Metadata in Polyswarm.')
+def metadata():
+    pass
+
+
+@metadata.command('rerun', short_help='Rerun metadata evaluation for the given hashes.')
+@click.option('-a', '--analysis', type=click.STRING, multiple=True,
+              help='Analysis to be execute on the Artifact.')
+@click.option('-s', '--skip-es', type=click.BOOL, is_flag=True,
+              help='Skip the upload to ES when evaluating the new metadata value.')
+@click.argument('hashes', nargs=-1, required=True)
+@click.pass_context
+def create(ctx, analysis, skip_es, hashes):
```

Comment:
if "-a" can take multiple strings, how does this know which is the last value that goes with "-a" and the first "hash" value?

Reply:
should we just take one hash, or use a separate arg to provide a file containing a list of hashes?